frontend/packages/kubevirt-plugin/src/components/vm-like/vm-like-resource-link.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import * as classNames from 'classnames';
+import { connectToModel } from '@console/internal/kinds';
+import { ResourceIcon, resourcePath } from '@console/internal/components/utils';
+import { getName, getNamespace } from '@console/shared';
+import { VirtualMachineInstanceModel, VirtualMachineModel } from '../../models';
+import { isVM } from '../../selectors/vm';
+
+export const VMLikeResourceLink = connectToModel(({ className, inline = false, title, obj }) => {
+  const name = getName(obj);
+  const namespace = getNamespace(obj);
+
+  const VMpath = resourcePath(VirtualMachineModel.kind, name, namespace);
+  const classes = classNames('co-resource-item', className, {
+    'co-resource-item--inline': inline,
+  });
+
+  return (
+    <span className={classes}>
+      <ResourceIcon
+        kind={isVM(obj) ? VirtualMachineModel.kind : VirtualMachineInstanceModel.kind}
```

Comment:
you could use `getVMLikeModel` here if we enhance `VMLikeEntityKind`

Reply:
done